README.md

```diff
@@ -32,12 +32,13 @@ npm install
 npm run start
 ```
 
-*But...* You need to run API Too, it's easy just:
+*But...* You also need to run an API server. It's easy! Just do the following:
 
-**4. Clone your fork of bladey-api**
+**4. Clone your fork of the bladey-api ([which you can find here](https://github.com/SwitchbladeBot/bladey-api))**
```

Comment:
Abaixo já tem o link, não vejo necessidade de repeti-lo.

Reply:
concordo

Reply:
nope, o link seria do fork.

Reply:
now lets talk england bois
We have two options:
- Leave it like it is (with the original repo url)
- Change the below link to the original url and the user will have to create a new remote manually to their fork (and the top line would be "Clone the bladey-api" instead of "Clone your fork of the bladey-api")

Reply:
Umm I think its best to leave it as it is